include/wlr/backend.h

```diff
@@ -11,16 +11,30 @@ struct wlr_backend {
 	const struct wlr_backend_impl *impl;
 
 	struct {
+		/** Raised when a wlr_input_device is added. */
 		struct wl_signal input_add;
```

Comment:
We should probably document for all `wl_list` and `wl_signal` which are the datatypes used.